modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceconfiguration/ServiceConfigurationQueryModel.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.database.serviceconfiguration;
+
+import java.util.UUID;
+import lombok.Data;
+
+/**
+ * The query model for search ServiceConfigurationEntity.
+ */
+@Data
+public class ServiceConfigurationQueryModel {
+
+    private UUID id;
```

Comment:
Removed it same time as deleting the medthod listServiceConfigurations(ServiceConfigurationQueryModel configQuery)